### PR TITLE
Add overlay server to testplanet

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: internal/testplanet/node.go [hunk 1]
```diff
@@ -73,9 +75,22 @@ func (node *Node) DialPointerDB(destination *Node, apikey string) (pdbclient.Cli
 		}
 		return piececlient.NewPSClient
 	*/
+
+	// TODO: handle disconnect
 	return pdbclient.NewClient(node.Identity, destination.Addr(), apikey)
 }
 
+// DialOverlay dials destination with apikey and returns pointerdb Client
+func (node *Node) DialOverlay(destination *Node) (pb.OverlayClient, error) {
```
Comment (kaloyan-raev) [inline internal/testplanet/node.go]: Can we have this method returning the `overlay.Client` interface? This is what, for example, the Segment Store expects as param

File: internal/testplanet/node.go [hunk 2]
```diff
@@ -73,9 +75,22 @@ func (node *Node) DialPointerDB(destination *Node, apikey string) (pdbclient.Cli
 		}
 		return piececlient.NewPSClient
 	*/
+
+	// TODO: handle disconnect
 	return pdbclient.NewClient(node.Identity, destination.Addr(), apikey)
 }
 
+// DialOverlay dials destination with apikey and returns pointerdb Client
```
Comment (kaloyan-raev) [inline internal/testplanet/node.go]: This comment seems to be copied from `DialPointerDB` and is not updated accurately.

File: internal/testplanet/node.go (hunk 1 above)
Comment (egonelbre) [reply internal/testplanet/node.go]: sounds good

File: internal/testplanet/node.go (hunk 2 above)
Comment (egonelbre) [reply internal/testplanet/node.go]: fixed

**Pull request merged**